src/Symfony/Component/Notifier/Bridge/MicrosoftTeams/composer.json

```diff
@@ -0,0 +1,30 @@
+{
+    "name": "symfony/microsoftteams-notifier",
```

Comment:
```suggestion
    "name": "symfony/microsoft-Teams-notifier",
```

Reply:
i think the uppercase on Teams is a typo mistake